provider/EmailAndPhone.iml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@shriharshs are you sure this file is needed? seems like an IDE file which may not need to be committed.

Reply:
@kochhar Sometimes IDE throws error without this .iml file when you import the code. We faced this issue in Android as well, so thought I'll add it.